spec/forms/search_form_spec.rb

```diff
@@ -507,4 +507,50 @@
       end
     end
   end
+
+  context "when scope considers patients with school moves or vaccination records" do
```

Comment:
Took me a bit to understand the setup of this test here, I wonder if it would be clearer if we said that we're using the `PatientPolicy::Scope`? Would the following be correct?

```suggestion
  context "using PatientPolicy scope which includes patients with school moves or vaccination records" do
```